db/migrate/20160825190205_devise_create_users.rb

```diff
@@ -0,0 +1,41 @@
+class DeviseCreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ''
+      t.string :encrypted_password, null: false, default: ''
```

Comment:
Resolved.

Looked at devise - this is the default model but the actual library calls Devise::Encryptor 
https://github.com/plataformatec/devise/blob/master/lib/devise/encryptor.rb
The function is hashing and salting the password but stores it as 'encrypted' - believe its just using the bcrypt password hashing function, not doing any encryption.